cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -28,1416 +47,1446 @@ def slice_records(self, output_records, expected_records, start_index=None, end_
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 15,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Miscellaneous",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 16,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Personal injury",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 17,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Public",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 18,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Generic",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 1,
-                "model": u"knowledgebase.article",
-                "fields": {
-                    "created": u"{date}".format(date=self.datetime_now),
-                    "modified": u"{date}".format(date=self.datetime_now),
-                    "resource_type": u"LEGAL",
-                    "website": u"https://www.google.com",
-                    "geographic_coverage": u"Baz",
-                    "type_of_service": u"Baz",
-                    "description": u"Bar",
-                    "service_name": u"Bar",
-                    "organisation": u"Foo",
-                    "helpline": u"Bar",
-                    "accessibility": u"Foo",
-                    "when_to_use": u"Baz",
-                    "how_to_use": u"Foo",
-                    "address": u"Foo",
-                    "keywords": u"Bar",
-                    "opening_hours": u"Bar",
-                },
-            },
-        ]
-        outputList = json.loads(outputJSON)
-        self.compare_length(outputList, expectedList)
-        self.compare_list_of_records(outputList, expectedList)
+        csv_file = self.find_path_to_csvfile("legal_resource_entry_type")
+        file = open(csv_file)
+        try:
+            csv = KnowledgebaseCsvParse(file)
+        except SystemExit:
+            self.fail("Missing required fields")
```

Comment:
How do you know this is the case from the exception? Seems like a risky error message